app/src/main/res/layout/screen_edit_patient.xml

```diff
@@ -200,6 +200,36 @@
         android:visibility="gone"
         tools:visibility="visible" />
 
+      <LinearLayout
+        android:id="@+id/addNHIDButtonContainer"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/spacing_8"
+        android:orientation="horizontal"
+        android:visibility="gone">
+
+        <ImageButton
+          android:id="@+id/addNHIDButton"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:padding="@dimen/spacing_2"
+          android:background="?attr/selectableItemBackgroundBorderless"
+          app:srcCompat="@drawable/ic_add_circle_outline_24px"
+          tools:ignore="ContentDescription" />
```

Comment:
Correct me if I am wrong, we want the entire container to be clickable right? Couldn't we have just used a borderless button with an icon? 🤔

Reply:
Even if it's not a button, I think we should handle the click from entire container rather than just the icon button, can you check with design once about this, please?

Reply:
@msasikanth this is how we have currently implemented both the buttons. I'll raise another pr to make [this](https://app.shortcut.com/simpledotorg/story/6797/make-the-entire-nhid-bp-passport-add-button-clickable-and-use-a-borderless-button-with-an-icon) change.

Reply:
Cool, thanks for creating the story for it 👍🏾